app/app.go

```diff
@@ -587,14 +583,14 @@ func New(
 	// 	consensusRegistry,
 	// )
 
-	// app.EvmKeeper = *evmmodulekeeper.NewKeeper(
-	// 	appCodec,
-	// 	keys[evmmoduletypes.StoreKey],
-	// 	memKeys[evmmoduletypes.MemStoreKey],
-	// 	app.GetSubspace(evmmoduletypes.ModuleName),
-	// 	app.ConsensusKeeper,
-	// 	app.ValsetKeeper,
-	// )
+	app.EvmKeeper = *evmmodulekeeper.NewKeeper(
+		appCodec,
+		keys[evmmoduletypes.StoreKey],
```

Comment:
Use `runtime.NewKVStoreService`  instead of StoreKey and update keeper to use StoreService
ref: https://github.com/cosmos/cosmos-sdk/blob/v0.50.1/UPGRADING.md#module-wiring